server/README.md

```diff
@@ -1,156 +1,31 @@
-```
-
-Alternatively, you can run the `data` part of this project and generate this file by that part of our docs.
-To link the output directory to the server data directory, so that you don't need to copy on every update you can use:
-
-```bash
-ln -s ../data/output data
-```
-
-##### Setting up the database
+Most of these components are independent of each other and share very few things.
+The only some api endpoints share is the database, which contains the building data.
 
 To set up the database, you will need to run the `load_api_data_to_db.py` script:
 
 ```bash
-python3 load_api_data_to_db.py
-```
-
-#### How to Set up the tileserver (needed for the `preview` endpoint)
-
-To set up your tileserver, head over to the [`map`](https://github.com/TUM-Dev/NavigaTUM/tree/main/map) folder and follow the instructions there.
-
-#### How to Set up MeiliSearch (needed for the `search` endpoint)
-
-The server uses [MeiliSearch](https://github.com/meilisearch/MeiliSearch) as a backend for search.
-For a local test environment you can skip this step if you don't want to test or work on search.
-
-There are a lot of different ways to run MeiliSearch (see on their repo). Here we compile it
-from sources:
-
-```bash
-# Clone MeiliSearch
-cd ..
-git clone https://github.com/meilisearch/MeiliSearch.git -b v0.22.0
-cd MeiliSearch
-
-# Build and run
-cargo run --release
-```
-
-Next, we need to add our index and configure search:
-
-```bash
-# Create index
-curl -i -X POST 'http://localhost:7700/indexes' --header 'content-type: application/json' --data '{ "uid": "entries", "primaryKey": "ms_id" }'
-
-# Set filterable attributes
-curl -X PUT 'http://localhost:7700/indexes/entries/settings/filterable-attributes' --data '["facet"]'
-
-# Upload entries data
-curl -i -X PUT 'http://localhost:7700/indexes/entries/documents' --header 'content-type: application/json' --data-binary @data/search_data.json
-
-# Configure index
-curl -X PUT 'http://localhost:7700/indexes/entries/settings/ranking-rules' --data '["words","typo","rank:desc","exactness","proximity","attribute"]'
-
-curl -X PUT 'http://localhost:7700/indexes/entries/settings/synonyms' --data @../data/search_synonyms.json
-
-curl -X PUT 'http://localhost:7700/indexes/entries/settings/searchable-attributes' --data '[ "ms_id", "name", "arch_name", "type", "type_common_name", "parent_building", "parent_keywords", "address", "usage" ]'
-```
-
-If you want to update the data in the index, run:
-
-```bash
-curl -i -X PUT 'http://localhost:7700/indexes/entries/documents' --header 'content-type: application/json' --data-binary @data/search_data.json
-```
-
-And if you want to delete the index, run:
-
-```bash
-curl -X DELETE 'http://localhost:7700/indexes/entries'
-```
-
-MeiliSearch provides an interactive interface at [http://localhost:7700](http://localhost:7700).
-
-### API-Changes
+ln -sf ../data/output/ data
+cd calendar && ln -sf ../../data/output/ data
+cd main-api && ln -sf ../data/output/ data
```

Comment:
```suggestion
cd calendar && ln -sf ../../data/output/ data && cd ..
cd main-api && ln -sf ../../data/output/ data
```
Shouldn't it be like this? Or do we need `data` mapped into the root `server` folder as well

Reply:
I am going to push this to another PR, because this way it stays reviewable